client/vscode/src/settings/endpointSetting.ts

```diff
@@ -22,33 +32,18 @@ export function endpointProtocolSetting(): string {
     return new URL(endpointSetting()).protocol
 }
 
-export function endpointAccessTokenSetting(): boolean {
-    if (readConfiguration().get<string>('accessToken')) {
-        return true
-    }
-    return false
-}
-
 export function endpointRequestHeadersSetting(): object {
     return readConfiguration().get<object>('requestHeaders') || {}
 }
 
-export async function updateEndpointSetting(newEndpoint: string, newAccessToken?: string): Promise<boolean> {
-    const newEndpointURL = removeEndingSlash(newEndpoint)
-    try {
-        if (newAccessToken) {
-            await readConfiguration().update('accessToken', newAccessToken, vscode.ConfigurationTarget.Global)
-        }
-        await readConfiguration().update('url', newEndpointURL, vscode.ConfigurationTarget.Global)
-        return true
-    } catch {
-        return false
-    }
-}
-
 function removeEndingSlash(uri: string): string {
     if (uri.endsWith('/')) {
         return uri.slice(0, -1)
     }
     return uri
 }
+
+export function isSourcegraph(): boolean {
```

Comment:
```suggestion
export function isSourcegraphDotCom(): boolean {
```

I think this is a bit clearer, wdyt?